### PR TITLE
Telegram notification

### PR DESCRIPTION
**Pull request by g-flex**
Instructions: 

Create a new bot (https://core.telegram.org/bots/api) with Botfather.
To retrieve your telegram_id, send a message to the bot, then open https://api.telegram.org/bot(your_bot_token)/getUpdates . You will find your telegram_id under message->from->id.
You are now able to fill all the properties in utils/Telegram.js and send messages to yourself.
Uncomment Telegram.init().telegramPost(r) in buy() function.
You will now receive a Telegram message as purchase notification!

### CODE REVIEW

File: utils/CoinbaseHelper.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ class CoinbaseHelper {
             };
             authedClient.placeOrder(buyParams).then(r => {
                 console.log(r);
+		        //Telegram.init().telegramPost(r);
```
Comment (paolo96) [inline utils/CoinbaseHelper.js]: Make a variable in Settings.js named isTelegramNotificationOn
Check with an if statement and remove the comment

File: utils/Telegram.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
+const fetch = require("node-fetch");
+
+class Telegram {
+
+    constructor() {
+
+        this.bot_token = "your_bot_token";
+        this.telegram_id = "00000000";
```
Comment (paolo96) [inline utils/Telegram.js]: Change to "your_telegram_id"

**Review by paolo96 — CHANGES_REQUESTED** [review]
Make these quick changes and I'll approve it

Also change the readme.md file to include instructions on how to set up the telegram bot (similar to what you did in the pull request comment)

**Comment by g-flex** [comment]
Done

**Pull request merged**